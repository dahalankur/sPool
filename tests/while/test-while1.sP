# simple while statement
while (true):
    x = x + 1
;