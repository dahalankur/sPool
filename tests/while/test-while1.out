PROGRAM {[ 
    WHILE(BOOL(true), [ 
        ASSIGN(x, BINOP(VAR(x), PLUS, INT(1)))])]}
